9/README.md

```diff
@@ -0,0 +1,325 @@
+```abnf
+ encoded-value = encoded_format encoded_data
+                 ; elements order implied by the above rule MAY be ignored
+```
+;where \<encoded_format\> has type DevString and \<encoded_data\> has type DevVarCharArray. 
+
+DevVarLongStringArray and DevVarDoubleStringArray values are structures to carry sequences of numbers 
+(integer or double precision floating point, respectively) along with sequences of text/string data.
+
+* Names DevVarLongStringArray and DevVarDoubleStringArray SHALL follow the following rules:
+```abnf
+ DevVarLongStringArray = "DevVarLongStringArray"
+ DevVarDoubleStringArray = "DevVarDoubleStringArray"
+```
+
+* Values of DevVarLongStringArray and DevVarDoubleStringArray SHALL follow rules \<long-string-value\> 
+and \<double-string-value\>, respectively:
+```abnf
+ long-string-value = lvalue svalue 
+ double-string-value = dvalue svalue
+                       ; elements order implied by the above rules MAY be ignored
+```
+;where \<lvalue\> has type DevVarLongArray, \<dvalue\> has type DevVarDoubleArray and \<svalue\> has type 
+DevVarStringArray.
+
+#### State
+
+DevState type standardizes a way of describing state of Device. Its values are from predefined set of 14 names.   
+
+* DevState name SHALL be according to the following rule:
+```abnf
+ DevState = "DevState"
+```
+
+* DevState type SHALL be an enumeration type with labels following the rule \<dev-state-label\>:
+
+```abnf
+ dev-state-label = "ALARM" / "INSERT" / "STANDBY" / "CLOSE" / "MOVING" / "UNKNOWN" / "DISABLE" / "OFF"
+ dev-state-label =/ "EXTRACT" / "ON" / "FAULT" / "OPEN" / "INIT" / "RUNNING"
+```
+
+#### Enumeration
+
+DevEnum data type allows to assign string Labels to DevShort values. It is valid only for Attributes. 
+
+* DevEnum name SHALL follow the following rule:
+```abnf
+ DevEnum = "DevEnum"
+```
+
+* Label values of DevEnum SHALL follow the rule \<devenum-label\>:
+```abnf
+ devenum-label = 1*VCHAR
+```  
+
+* DevEnum labels SHALL be unique in the context of an Attribute.
+* Each of DevEnum labels SHALL correspond to one and unique DevShort value. 
+* Related DevShort values SHALL be consecutive and SHALL start with 0. 
+* Values of DevEnum DataType SHALL be transported as values of DevShort.
+
+* For any Attribute of DevEnum data type, list of Labels SHALL be available as `enum_labels` Attribute property (see [4/Attribute](/4)).
+
+* Tango Controls SHALL allow a Tango Client to retrieve labels associated with the DevShort value.
+
+#### Pipe
+
+DevPipeBlob is a Data Type to transfer data related to Pipes (see 7/Pipe).
+
+* DevPipeBlob SHALL have the following name:
+```abnf
+ DevPipeBlob = "DevPipeBlob"
+```
+
+* A value of DevPipeBlob type SHALL follow the \<pipe-blob-value\> rule:
+
+```abnf
+ pipe-blob-value = name blob-data
+                   ; elements order implied by the above rule MAY be ignored
+
+ name = 1*VCHAR 
+
+ blob-data = *blob-data-element
+ 
+ blob-data-element = name value inner-blob inner-blob-name 
```

Comment:
I think it is not clear what `value ` is here.  (Scalar or array of any basic Tango type)
I also think inner-blob-name does not make sense when the inner-blob is empty.

Reply:
@bourtemb, this [commit](https://github.com/tango-controls/rfc/pull/61/commits/5fb3d9dce38176bc6b4339f208959f8f8b7f6df3) adds the missing `value` specification.

Reply:
Regarding the `inner-blob-name`, even it has no meaning in case of inner-blob is empty, the IDL specifies it as a part of the [DevPipeDataElt structure](https://github.com/tango-controls/tango-idl/blob/1e5edb84d966814ad367f2674ac9a5658b6724ac/tango.idl#L555).